Formula/dillo.rb

```diff
@@ -0,0 +1,45 @@
+class Dillo < Formula
+  desc "Graphical web browser known for its speed and small footprint"
+  homepage "http://www.dillo.org/"
+  url "http://www.dillo.org/download/dillo-3.0.5.tar.bz2"
+  sha256 "db1be16c1c5842ebe07b419aa7c6ef11a45603a75df2877f99635f4f8345148b"
+
+  depends_on "openssl"
+  depends_on "fltk"
+
+  # for errors about Fl_Font.H (fixed in HEAD)
+  patch :DATA
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-ssl",
+                          "--prefix=#{prefix}"
+    system "make", "install"
```

Comment:
Like elsewhere: if it passes locally for you on macOS and takes <1m then it seems OK.